rules/falco_rules.yaml

```diff
@@ -1230,10 +1230,14 @@
   tags: [maturity_stable, host, container, network, process, mitre_execution, T1059]
 
 - list: known_memfd_execution_binaries
-  items: []
+  items: [runc]
 
 - macro: known_memfd_execution_processes
-  condition: (proc.name in (known_memfd_execution_binaries))
+  condition: > 
+    (proc.name in (known_memfd_execution_binaries))
```

Comment:
Hi there! thanks for opening the PR. 
I am not sure though this will work since:
- when we execute from memfd, usually we use the proc filesystem reference to the fd to do so. The process name will be a numeric value such as "3"
- IIRC, Falco doesn't resolve exepath and exe that way
Possibly, the best option would be to tune the rule by using ancestors information, such as proc.pname and proc.aname